src/models/activityReportCollaborator.js

```diff
@@ -0,0 +1,24 @@
+import { Model } from 'sequelize';
+
+export default (sequelize, DataTypes) => {
+  class ActivityReportCollaborator extends Model {
+    static associate(models) {
+      ActivityReportCollaborator.belongsTo(models.ActivityReport, { foreignKey: 'activityReportId', as: 'activityReport' });
+      ActivityReportCollaborator.belongsTo(models.User, { foreignKey: 'userId', as: 'user' });
+    }
+  }
+  ActivityReportCollaborator.init({
+    activityReportId: {
+      allowNull: false,
+      type: DataTypes.INTEGER,
+    },
+    userId: {
+      allowNull: true,
```

Comment:
Could we have a record in the ActivityReportCollaborator table without a user?

Reply:
good catch, I'll update